DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Dashboard/RankingController.cs

```diff
@@ -19,7 +19,7 @@ public RankingController(ICentresService centresService)
             this.centresService = centresService;
         }
 
-        public IActionResult Index()
+        public IActionResult Index(int? regionId = null)

```

Comment:
Unrelated to ticket - intentional?

Reply:
Yeah accidental change from a different ticket